changelog.md

```diff
@@ -1,3 +1,11 @@
+### v4.2.9
```

Comment:
@matislovas I think it would be smart to wait for @tomaszklak changes, and release together with it

Reply:
I think he wanted to do release later anyway, I just bumped changelog version in advance, so another PR wouldn't be needed if it was decided to release now

Reply:
But now I see that Cargo.toml wasn't changed anyway :smile: